app/templates/components/curriculum-inventory-report-rollover.hbs

```diff
@@ -0,0 +1,56 @@
+<div class='rollover-form'>
+  <h3 class="title">{{t 'general.curriculumInventoryReportRollover'}}</h3>
+
+  <p class='rollover-summary'>{{t 'general.curriculumInventoryReportRolloverSummary'}}</p>
+
+  <div class='item name'>
+    <label>{{t 'general.name'}}:</label>
+    {{one-way-input
+      value=name
+      update=(pipe (action 'changeName') (action 'addErrorDisplayFor' 'name'))
```

Comment:
Can this be
`update=(pipe (action (mut changeName) (action 'addErrorDisplayFor' 'name'))`? And then remove the changeName action from the components?  I'm not sure how `pipe` would work in that case.